ldap/admin/src/logconv.pl

```diff
@@ -355,6 +355,7 @@
 $connmsg{"B4"} = "Server failed to flush data (response) back to Client";
 $connmsg{"T1"} = "Idle Timeout Exceeded";
 $connmsg{"T2"} = "IO Block Timeout Exceeded or NTSSL Timeout";
+$connmsg{"T3"} = "Paged Search Limit Exceeded";
```

Comment:
I thought it was best to another another T version, for T2 there are already 2 timeout reasons.

Reply:
Right good idea. I would replace 'Search Limit' with 'Search Time Limit'